src/pks/transport/transport_ats_pk.cc

```diff
@@ -1520,6 +1552,12 @@ void Transport_ATS::ComputeAddSourceTerms(double tp, double dtp,
   int num_vectors = cons_qty.NumVectors();
   int nsrcs = srcs_.size();
 
+  if (trans_srs_key_!=""){
+    if (S_->HasEvaluator(trans_srs_key_, Tags::NEXT)) {
+      S_->GetEvaluator(trans_srs_key_, Tags::NEXT).Update(*S_, trans_srs_key_);
```

Comment:
I'm confused -- I see you requiring this key, and updating it, but I don't see it actually used anywhere?

Reply:
I removed it. Evaluator is updated within domain function.